groups/infrastructure/alb.tf

```diff
@@ -0,0 +1,26 @@
+module "alb" {
+  source = "git@github.com:companieshouse/terraform-modules//aws/application_load_balancer?ref=1.0.205"
+
+  environment         = var.environment
+  service             = local.service_name
+  ssl_certificate_arn = aws_acm_certificate.certificate.arn
+  subnet_ids          = data.aws_subnets.private.subnet_ids
```

Comment:
`subnet_ids` isn't an exported attribute for this data source. This should be `ids`